app/main.py

```diff
@@ -1,8 +1,50 @@
+from __future__ import annotations
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
Place your arguments in the `__init__` from a new line each, so that opening and closing brackets will not have arguments.